tests/test_backend/test_mlir/golds/simple_structural_location_info_style_wrapInAtSquareBracket.mlir

```diff
@@ -1,5 +1,5 @@
 module attributes {circt.loweringOptions = "locationInfoStyle=wrapInAtSquareBracket"} {
-    hw.module @simple_structural(%a: i16, %b: i16, %c: i16, %CLK: i1) -> (y: i16, z: i16) {
+    hw.module @simple_structural(%a: i16 loc(unknown), %b: i16 loc(unknown), %c: i16 loc(unknown), %CLK: i1 loc(unknown)) -> (y: i16 loc(unknown), z: i16 loc(unknown)) {
```

Comment:
Should we include a test that has a known location?

Reply:
we currently don't ever emit valid locations for ports. we need to add that feature; noted in the PR